packages/scanner/src/page.spec.ts

```diff
@@ -216,7 +216,7 @@ describe('Page', () => {
 
         await page.enableBypassCSP();
 
-        const results = await page.scanForA11yIssues('https://www.bing.com');
+        const results = await page.scanForA11yIssues('https://tinyurl.com/mrhd5x');
```

Comment:
why do we have to change the url in this test case?

Reply:
My mistake, I was testing with tinyurl and forgot to change it back. That's fixed now.